dGame/UserManager.cpp

```diff
@@ -451,13 +451,13 @@ void UserManager::RenameCharacter(const SystemAddress& sysAddr, Packet* packet)
 		"User %i tried to rename a character that it does not own!",
 		u->GetAccountID());
 
-	std::find_if(u->GetCharacters().begin(), u->GetCharacters().end(), [&](Character* c) {
+	static_cast<void>(std::find_if(u->GetCharacters().begin(), u->GetCharacters().end(), [&](Character* c) {
```

Comment:
Just capture the return value, I believe this reads weird and it would be better to just capture instead of casting to void like this.

Reply:
This just isn't syntaxes we use and it looks clunky to a reader, we generally just follow a pattern of catching the return value.  If you absolutely must, rework the find_if to do the same work and capture the return value like that.  Note that a cast to void does not always dismiss the return value from a function in some cases.  It's a pretty old syntax that I don't see used in modern day.

Reply:
What about std::ignore? I just want to make it clear the return value is being discarded, because an 'auto' is ambiguous.

Reply:
Std ignore is not intended for use outside of std tuple and std tie.  The current pattern in the codebase is to just capture the return value and do nothing with it (we do this in other places without issue).  If you don't like the auto you could use the verbose name but it'd be just easier to capture it and do nothing.  If you must, a future pr could enable the warning for unused local variables.

Reply:
I seem to be in the minority with my opinion on the use of casting to void, however even knowing so I believe it is better to re-write this code to use the return value than to discard it since this code could use the iterator anyways.

Reply:
Updated